docs/Documentation-CHN/UserGuide/8-Tools-watermark.md

```diff
@@ -0,0 +1,203 @@
+|      1970-01-01T08:00:00.007+08:00|               107|              null|
+|      1970-01-01T08:00:00.008+08:00|               108|              null|
+|      1970-01-01T08:00:00.009+08:00|               109|              null|
+|      1970-01-01T08:00:00.010+08:00|               110|              null|
+|      1970-01-01T08:00:00.011+08:00|               111|              null|
+|      1970-01-01T08:00:00.012+08:00|               112|              null|
+|      1970-01-01T08:00:00.013+08:00|               113|              null|
+|      1970-01-01T08:00:00.014+08:00|               114|              null|
+|      1970-01-01T08:00:00.015+08:00|               115|              null|
+|      1970-01-01T08:00:00.016+08:00|               116|              null|
+|      1970-01-01T08:00:00.017+08:00|               117|              null|
+|      1970-01-01T08:00:00.018+08:00|               118|              null|
+|      1970-01-01T08:00:00.019+08:00|               119|              null|
+|      1970-01-01T08:00:00.020+08:00|               120|              null|
+|      1970-01-01T08:00:00.021+08:00|               121|              null|
+|      1970-01-01T08:00:00.022+08:00|               122|              null|
+|      1970-01-01T08:00:00.023+08:00|               123|              null|
++-----------------------------------+------------------+------------------+
+```
+
+<a id="%E7%AC%AC%E4%BA%8C%E6%AD%A5%EF%BC%9A%E7%BB%99alice%E6%96%BD%E5%8A%A0%E6%B0%B4%E5%8D%B0%E5%B5%8C%E5%85%A5"></a>
+#### 第二步：给Alice施加水印嵌入
+
+sql用法：`grant watermark_embedding to a,b` 
+
+只有root用户有权限运行该指令。在root给Alice施加水印嵌入之后，Alice的所有查询结果都将被嵌入水印。
+
+```
+.\start-client.bat -u root -pw root
+grant watermark_embedding to Alice
+exit
+
+.\start-client.bat -u Alice -pw 1234
+select * from root
+
++-----------------------------------+------------------+------------------+
+|                               Time|root.vehicle.d0.s0|root.vehicle.d0.s1|
++-----------------------------------+------------------+------------------+
+|      1970-01-01T08:00:00.001+08:00|               100|              null|
+|      1970-01-01T08:00:00.002+08:00|               102|              null|
+|      1970-01-01T08:00:00.003+08:00|               103|              null|
+|      1970-01-01T08:00:00.004+08:00|               104|               104|
+|      1970-01-01T08:00:00.005+08:00|               105|              null|
+|      1970-01-01T08:00:00.006+08:00|               106|              null|
+|      1970-01-01T08:00:00.007+08:00|               107|              null|
+|      1970-01-01T08:00:00.008+08:00|               108|              null|
+|      1970-01-01T08:00:00.009+08:00|               109|              null|
+|      1970-01-01T08:00:00.010+08:00|               110|              null|
+|      1970-01-01T08:00:00.011+08:00|               111|              null|
+|      1970-01-01T08:00:00.012+08:00|                96|              null|
```

Comment:
It would be nice if you bold (I do not know if it works in a table) what has been changed in two table.
And, this is only my soliloquy, if I were the user, I might get frightened by this example because it seems to have changed some data dramatically. I think an example with slighter data change would be less terrifying (like a big float number with only its tail is changed).

Reply:
Good suggestion! fixed.